proof-new/exec.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I would remove this script.  This will eventually be covered as part of our regression test script (https://github.com/cvc5/cvc5/blob/main/test/regress/cli/run_regression.py).

Reply:
deleted